datasets/us/ak/med_exclusions/crawler.py

```diff
@@ -0,0 +1,53 @@
+from typing import Dict
+from rigour.mime.types import PDF
+
+from zavod import Context, helpers as h
+
+from zavod.shed.gpt import run_image_prompt
+
+prompt = """
+Extract structured data from the following page of a PDF document. Return 
+a JSON list (`providers`) in which each object represents an medical provider.
+Each object should have the following fields: `exclusion_date`, `last_name`,
```

Comment:
This seems to fix the example you showed. Does it fix any others that you found to have issues? Does it introduce any other issues?

```suggestion
a JSON list (`providers`) in which each object represents an medical provider.
Distinct record rows alternate between grey and white backgrounds. The name
of a provider might be split onto the next row within a record.
Each object should have the following fields: `exclusion_date`, `last_name`,
```

I am absolutely blown away that this works. So interesting.

Reply:
I was too! It almost look like magic.

Apparently this doesn't introduce any new issues, I ran and it didn't enter the `not row.get("last_name")` if statement.